### ISSUE TITLE
Calendar entries from Baikal dont show up when using Reverse Proxy

### ISSUE THREAD

**Issue by simonl169** [intri-in/manage-my-damn-life-nextjs#173]
Hi!

I am using MMDL together with Baikal as CalDav. I encountered a strange issue where the entries form Baikal dont show up in the Calendar when I acces MMDL via Nginx Proxy Managaer Reverse Proxy. **If I access the container directly via IP and Port, it works flawlessly.**
However, I can still create new entries which then show up on my second client (iPhone Standard Calendar App). Maybe this is more of a problem with the Reverse Proxy, but dunno

Here some more Infos:
Baikal on Docker, via Reverse Proxy
MMDL on Docker, via Reverse Proxy
Nginx Proxy Manager as Reverse Proxy
.well-knonw Redirects etc are set for Baikal on NPM, works fine with iPhone and Android
Using MariaDB as Database since I couldnt get mysql running

**Comment by intri-in** [comment]
This sounds like an issue with the reverse proxy configuration.

It's weird that you can make write requests, but not read requests.

When accessing through the Nginx proxy, can MMDL find your calendars?

Maybe the fetch request is being blocked at the proxy end? Can you check the logs of NPM and see if MMDL makes a request and what status code you find?

Also, when you add your Baikal account to MMDL, instead of using "http://yourdomain.example.com/" in Caldav server Url field, try using "http://yourdomain.example.com/dav.php" as the URL, and see if the issue resolves.

**Comment by simonl169** [comment]
> When accessing through the Nginx proxy, can MMDL find your calendars?
Yes, I can find the calendars and even set it up as a new account
>Maybe the fetch request is being blocked at the proxy end? Can you check the logs of NPM and see if MMDL makes a request and what status code you find?
Working on that
>Also, when you add your Baikal account to MMDL, instead of using "http://yourdomain.example.com/" in Caldav server Url field, try using "http://yourdomain.example.com/dav.php" as the URL, and see if the issue resolves.
Yes, I am already using this, the other option would not work

**Comment by simonl169** [comment]
> [06/May/2024:21:12:23 +0200] - 207 207 - PROPFIND https yourdomain.com "/dav.php/principals/User1/" [Client 192.168.176.1] [Length 464] [Gzip -] [Sent-to 192.168.42.107] "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)" "-"
> [06/May/2024:21:12:23 +0200] - 207 207 - PROPFIND https yourdomain.com "/dav.php/calendars/User1/" [Client 192.168.176.1] [Length 3149] [Gzip -] [Sent-to 192.168.42.107] "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)" "-"
> [06/May/2024:21:12:23 +0200] - 207 207 - PROPFIND https yourdomain.com "/dav.php/calendars/User1/default/" [Client 192.168.176.1] [Length 1118] [Gzip -] [Sent-to 192.168.42.107] "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)" "-"
> [06/May/2024:21:12:23 +0200] - 207 207 - PROPFIND https yourdomain.com "/dav.php/calendars/User1/NewCalendar/" [Client 192.168.176.1] [Length 1117] [Gzip -] [Sent-to 192.168.42.107] "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)" "-"

This is the log after I click "Sync" in the menu, does seem okay

**Comment by intri-in** [comment]
Interesting. The PROPFIND requests are there, but there are no REPORT requests for the actual objects (events/tasks). 

I will investigate this, meanwhile perhaps you can try MMDL with the IP address directly.
